src/resources/generated/issuing_card.rs

```diff
@@ -608,7 +608,6 @@ pub enum IssuingCardSpendingLimitCategories {
     HardwareStores,
     HealthAndBeautySpas,
     HearingAidsSalesAndSupplies,
-    #[serde(rename = "heating_plumbing_a_c")]
     HeatingPlumbingAC,
```

Comment:
Seems like this case is correct. See https://stripe.com/docs/issuing/categories

Reply:
What do you mean by that? Since the changes removed the rename it was deemed not necessary, serde already renames it to `heating_plumbing_a_c` via `rename_all = "snake_case`.